perception/autoware_tensorrt_bevdet/README.md

```diff
@@ -0,0 +1,69 @@
+# tensorrt_bevdet <!-- cspell: ignore bevdet -->
+
+## Purpose
+
+tensorrt_bevdet is a dynamic 3D bev object detection package based on 6 surround view cameras.
+
+## Inner-workings / Algorithms
```

Comment:
The description of BEVDet is good, but could benefit from mentioning the specific advantages of using TensorRT (e.g., improved inference speed, lower latency).  Highlighting these performance aspects would be beneficial.